src/services/llms/gemini.py

```diff
@@ -28,6 +28,7 @@ def create_prompt(self, file: PatchedFile, hunk: Hunk, pr_details: PRDetails) ->
         Review the following code diff in the file "{file.path}" and take the pull request title and description into account.
         Pull request title: {pr_details.title}
```

Comment:
Pull request description is missing. Please add a description explaining the changes made and the reasoning behind them.  This will help reviewers understand the context and purpose of the changes.